src/colorschemes.jl

```diff
@@ -187,17 +187,13 @@ If `rev` is `true` colors are reversed.
 If `alpha` is set, it is applied to all colors.
 """
 function cgrad(
-    colors,
+    colors::ColorScheme,
```

Comment:
Now the tests fail with
```
MethodError: no method matching cgrad(::PlotUtils.ContinuousColorGradient)
```
Basically `colors` could be anything that can be handled by `get_colorscheme`, which is `AbstractColorList` (a parent type of colorgradients and palettes), `AbstractVector`, `Symbol` and `Colorant`.